{{cookiecutter.project_dashed}}-src/setup.cfg

```diff
@@ -0,0 +1,18 @@
+# Errors: https://lintlyci.github.io/Flake8Rules/
+#
+#
+# This is just the default list of ignores which you can get by looking at flake8 --help
+#
+# E121 - A line is less indented than it should be for hanging indents.
+# E123 - Closing brackets should match the same indentation level of the line that their opening
+#        bracket started on.
+# E126 - A continuation line is indented farther than it should be for a hanging indent.
+# E226 - There should be one space before and after an arithmetic operator (+, -, /, and *).
+# E241 - There should be only one space after the , character.
+# E704 - Multiple statements of a function definition should be on their own separate lines.
+# W503 - Line breaks should occur after the binary operator to keep all variable names aligned.
+# W504 - Line breaks should occur before the binary operator to keep all operators aligned.
+[flake8]
+exclude=.git,.hg,.tox,dist,doc,*egg,build
+ignore=E121,E123,E126,E226,E24,E704,W503,W504
```

Comment:
Seems like we came up with a much smaller list in: https://github.com/level12/keg-app-cookiecutter/issues/57